python/py-cli-helpers/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-cli-helpers
```

Comment:
if you add `python.rootname cli_helpers` here then you can remove the `master_sites` and `distname` fields below